Training4/src/task2/src/App.js

```diff
@@ -0,0 +1,72 @@
+import "./App.css";
+import { useEffect, useState } from "react";
+import Country from "./Country";
+import { debounce } from "lodash";
+
+function App() {
+  const [searchTerm, setSearchTerm] = useState("");
+  const [listContries, setListContries] = useState([]);
+
+  const getAPI = (url) => {
+    console.log(url);
+    fetch(url)
+      .then((response) => response.json())
+      .then((data) => {
+        setListContries(data);
+      })
+  };
+
+  const searchCountry = () => {
+    if (searchTerm.length > 0) {
+      console.log(searchTerm);
+      getAPI(`https://restcountries.eu/rest/v2/name/${searchTerm}`);
+    } else getAPI("https://restcountries.eu/rest/v2/all");
+  };
+
+  const searchTermLimit = debounce(searchCountry, 1000);
```

Comment:
same with task 1

Reply:
searchCountry should take the searchTerm as function parameter, and use need to useEffect watch the searchTerm change and to to debouncedSearchCountry

Reply:
My solution: https://github.com/longhpzigvy/zv-react-training/pull/28/commits/43a523eac070e16bda50cb0b0cce9d03d1c79323#r715460603